frontend/sozisel-app/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
+    "downlevelIteration": true,
```

Comment:
what is it for?

Reply:
It is not needed now, It was required to get rid of some problem with `Set` type